src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.ts

```diff
@@ -429,19 +415,23 @@ async function getSuggestionsWithinCommandExpression(
   const references = { fields: fieldsMap, variables: anyVariables };
   if (commandDef.suggest) {
     // The new path.
-    return commandDef.suggest(
+    return commandDef.suggest({
       innerText,
       command,
       getColumnsByType,
-      (col: string) => Boolean(getColumnByName(col, references)),
-      () => findNewVariable(anyVariables),
-      (expression: ESQLAstItem | undefined) =>
+      columnExists: (col: string) => Boolean(getColumnByName(col, references)),
+      getSuggestedVariableName: () => findNewVariable(anyVariables),
```

Comment:
I think we need at some point to rename these variables to runtimeFields or userColumns or something like that. The term variable confuses me every time here (irrelevant with your PR)

Reply:
Yo, totally ++